src/xcode/ENA/ENA/Source/Scenes/FileScanner/FileScannerCoordinatorViewModel.swift

```diff
@@ -32,6 +37,8 @@ enum FileScannerError: CaseIterable {
 			return AppStrings.FileScanner.PasswordError.title
 		case .alreadyRegistered:
 			return AppStrings.FileScanner.AlreadyRegistered.title
+		default:
+			return ""
```

Comment:
Better add a `case . qrCodeParserError` - so we will get errors if the enum gets extended